core/worker.cc

```diff
@@ -298,7 +311,7 @@ void launch_worker(int wid, int core) {
 Worker *get_next_active_worker() {
   static int prev_wid = 0;
   if (num_workers == 0) {
-    launch_worker(0, FLAGS_c);
+    launch_worker(0, FLAGS_c, "fast");
```

Comment:
Perhaps we can make "fast" as a default value for the third function parameter, and not specify it here..?